openmetadata-service/src/main/java/org/openmetadata/service/util/IngestionPipelineUtils.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.openmetadata.service.util;
+
+import org.openmetadata.schema.entity.services.ingestionPipelines.AirflowConfig;
+
+public class IngestionPipelineUtils {
+  public static AirflowConfig getDefaultAirflowConfig() {
+    return new AirflowConfig()
+        .withPausePipeline(false)
+        .withConcurrency(1)
+        .withPipelineTimezone("UTC")
+        .withRetries(3)
+        .withRetryDelay(300)
+        .withPipelineCatchup(false)
+        .withScheduleInterval("0 0 * * *")
```

Comment:
are we hard-coding the schedule interval here?

Reply:
@harshach this is the default airflow config that we are using to create a insights and reindexing pipeline on startup , user can later on will be able to edit these if needed